src/gpuhunt/providers/gcp.py

```diff
@@ -43,6 +49,28 @@
 }
 accelerator_counts = [1, 2, 4, 8, 16]
 
+json_file_path = os.path.join(
+    os.path.dirname(__file__), "..", "..", "..", "..", "gpuhunt/resources", "tpu_pricing.json"
+)
```

Comment:
I believe tpu_pricing.json is not included in the gpuhunt package (it's outside src/). Ensure that tpu_pricing.json is available when you build and install gpuhunt. Also consider using `pkg_resources` to load package resources instead of traversing the file system like that.

Reply:
@r4victor I have included tpu_pricing.json in `src/gpuhunt/resources`. Is this location ok? Also I have used `pkg_resources ` to load package resources. But I found that pkg_resources will be deprecated in future versions [See](https://setuptools.pypa.io/en/latest/pkg_resources.html)

Reply:
The location is good. I wasn't aware `pkg_resources` is deprecated. Let's use `importlib.resources` then?